doc/sphinx/source/recipes/broken_recipe_list.rst

```diff
@@ -9,26 +9,56 @@ More details can be found in the :ref:`broken recipe policy
 <broken-recipe-policy>`.
 
 .. list-table:: Broken recipes
-   :widths: 25 25 25 25
+   :widths: 25 25 25 25 25
    :header-rows: 1
 
    * - Broken recipe
      - Affected diagnostics
+     - Broken since release
      - Problem
      - GitHub issue
-   * - `recipe_check_obs.yml`
-     - `ERA5_native6`
-     - Derivation of custom variables `rlus` and `rsus`
-     - `#1388 <https://github.com/ESMValGroup/ESMValCore/issues/1388>`_
-   * - :ref:`recipe_julia.yml <recipe_examples>`
-     - `example`
-     - fill values are not interpreted, resulting in an unusable plot
-     - `#2595 <https://github.com/ESMValGroup/ESMValTool/issues/2595>`_
-   * - :ref:`recipe_seaice_drift.yml <recipes_seaice_drift>`
-     - `sea_ice_drift_SCICEX`
-     - ``shapely 2`` issue
-     - `#3243 <https://github.com/ESMValGroup/ESMValTool/issues/3243>`_
-   * - :ref:`recipe_pysplot.yml <recipes_psyplot_diag>`
-     - `plot_map`
-     - ``shapely 2`` issue
-     - `#3483 <https://github.com/ESMValGroup/ESMValTool/issues/3483>`_
+   * - :ref:`recipe_climwip_brunner2019_med.yml <recipe_climwip>`
+     - All (preprocessor issue)
+     - v2.11.0
+     - Failed to run preprocessor function ``fix_metadata`` on the data: Unable to convert units
+     - `#3694 <https://github.com/ESMValGroup/ESMValTool/issues/3694>`_
+   * - :ref:`recipe_collins13ipcc.yml <nml_collins>`
+     - All (preprocessor issue)
+     - v2.10.0
+     - Failed to run preprocessor function ``save`` on the data: HDF error
```

Comment:
I'm not sure if this error is sufficient to classify a recipe as broken, see https://github.com/ESMValGroup/ESMValTool/issues/3702#issuecomment-2202608401. For the previous releases, I did not mark recipes that failed with this error as broken. They are untested though..

Reply:
this is not a recipe issue per se, but rather, HDF5 not being able to deal with threaded IO, we still have to understand what particular setup causes this, but it's been seen before (as both you guys point out), also it's been not 100% reproducible - I'd not put these recipes under the broken tab

Reply:
My efforts to test these recipes on JASMIN have been unsuccessful so far (https://github.com/ESMValGroup/ESMValTool/issues/3702#issuecomment-2206239337) 😞 Since both you and @valeriupredoi agree that these recipes should not be considered broken, and in favour of making progress with the release, I have removed these recipes from the broken recipes list: 50b822cbc

Reply:
cheers, Emma :beer: